plenum/server/node.py

```diff
@@ -1641,13 +1641,36 @@ def handleInvalidClientMsg(self, ex, wrappedMsg):
         if isinstance(msg, Request):
             msg = msg.as_dict
         identifier = idr_from_req_data(msg)
-        reqId = msg.get(f.REQ_ID.nm)
+        # we send reqId == 1 when we need to reply on invalid LEDGER_STATUS
+        reqId = msg.get(f.REQ_ID.nm) or 1
         if not reqId:
             reqId = getattr(exc, f.REQ_ID.nm, None)
             if not reqId:
                 reqId = getattr(ex, f.REQ_ID.nm, None)
         self.send_nack_to_client((identifier, reqId), reason, frm)
         self.discard(wrappedMsg, friendly, logger.info, cliOutput=True)
+        self._specific_invalid_client_msg_handling(ex, wrappedMsg)
+
+    def _specific_invalid_client_msg_handling(self, ex, wrappedMsg):
+        op = wrappedMsg[0].get('op')
+        if (op == LEDGER_STATUS):
+            self._invalid_client_ledger_status_handling(ex, wrappedMsg)
+
+    def _invalid_client_ledger_status_handling(self, ex, wrappedMsg):
```

Comment:
Please add a test for this (that we send LedgerStatus back if protocol version is incorrect)

Reply:
ok